lib/PolynomialTermOperations.js

```diff
@@ -22,7 +21,7 @@ PolynomialTermOperations.rearrangeCoefficient = function(node) {
   if (!canRearrangeCoefficient(node)) {
     return NodeStatus.noChange(node);
   }
-  const oldNode = clone(node, false);
+  const oldNode = clone(node);
```

Comment:
do you want to make the change so this is newNode in this module? or leave that for another time